File: youlexuan-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {

    this.findPage = function (pageNum,pageSize) {
        return $http.get("../brand/findPage.do?page="+pageNum+"&rows="+pageSize);
    }

    /**
     * @param pageNum
     * @param pageSize
     * 根据条件模糊查询
     */
   this.search = function (pageNum,pageSize,searchEntiy) {
        return $http.post("../brand/search.do?page="+pageNum+"&rows="+pageSize,searchEntiy);
    }

    this.save = function (entity) {
        var mname = "add.do";
        if(entity.id !=null ){
            mname = "update.do";
        }
        return $http.post("../brand/"+mname,entity );
    }

    this.findOne = function (id) {
        return $http.get("../brand/findOne.do?id="+id);
    }

    this.delet = function (selectIds) {
       return $http.get("../brand/delet.do?ids="+selectIds);
    }

    this.findBrandList =function () {
        return $http.get("../brand/findBrandList.do");
    }

})

File: youlexuan-service-interface/src/main/java/com/youlexuan/sellergoods/service/BrandService.java
package com.youlexuan.sellergoods.service;

import com.youlexuan.entity.PageResult;
import com.youlexuan.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {

    List<TbBrand> findAll();

    //分页查询
    PageResult findPage(TbBrand brand,int pageNum,int pageSize);

    void  insert(TbBrand brand);

    void update(TbBrand brand);

    TbBrand findOne(Long id);

    void delte(Long[] ids);

    public List<Map> selectOption();
}
